drivers/gles2/shader_compiler_gles2.cpp

```diff
@@ -911,7 +911,7 @@ ShaderCompilerGLES2::ShaderCompilerGLES2() {
 	actions[VS::SHADER_CANVAS_ITEM].renames["INSTANCE_CUSTOM"] = "instance_custom";
 
 	actions[VS::SHADER_CANVAS_ITEM].renames["COLOR"] = "color";
-	actions[VS::SHADER_CANVAS_ITEM].renames["MODULATE"] = "final_modulate";
+	actions[VS::SHADER_CANVAS_ITEM].renames["MODULATE"] = "FINAL_MODULATE_ALIAS";
```

Comment:
Small nitpick. For style consistency, this should still be lowercase.

Reply:
the `final_modulate_alias`? Yup I did wonder about that. Normally I tend to use upper case for macros but here lower case could be better.

Reply:
Fixed.